lua/nvim-tree/keymap.lua

```diff
@@ -69,6 +69,7 @@ function M.default_on_attach(bufnr)
   vim.keymap.set('n', 'g?',      api.tree.toggle_help,                opts('Help'))
   vim.keymap.set('n', 'gy',      api.fs.copy.absolute_path,           opts('Copy Absolute Path'))
   vim.keymap.set('n', 'H',       api.tree.toggle_hidden_filter,       opts('Toggle Filter: Dotfiles'))
+  vim.keymap.set('n', 'h',       api.tree.toggle_binaries_filter,       opts('Toggle Filter: Binaries'))
```

Comment:
I'm sorry, this is too niche a case to warrant its own filter and mapping.

`filters.custom` will work nicely to cover the binary case.

Reply:
Reverted 👍